assets/src/application/ui/display/loadingNotice/LoadingNotice.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Spin } from 'antd';
+import { _x, sprintf } from '@wordpress/i18n';
+
+import { LoadingNoticeProps } from './types';
+
+const LoadingNotice: React.FC<LoadingNoticeProps> = ({ children, loading, size, ...props }) => {
+	const className = classNames(props.className, 'ee-loading-notice', `ee-loading-notice-${size}`);
+	const tip = props.tip || sprintf(_x('loading%s', 'loading...', 'event_espresso'));
+	let loadingProp = loading;
```

Comment:
the same as here: https://github.com/ant-design/ant-design/blob/master/components/list/index.tsx#L184